docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: explain
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2047
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
```

Comment:
ah. nevermind. I see now that you're using `DEBUG` in the examples. Still, it seems that plain `EXPLAIN` would be fine.